src/vue/pages/Funds.vue

```diff
@@ -1,61 +1,403 @@
+        >
+          <div class="fund-card__header">
+            <img
+              class="fund-card__logo"
+              :src="fund.logoUrl(config.FILE_STORAGE)"
+            >
+          </div>
+
+          <div class="fund-card__info">
+            <p class="fund-card__name">
+              {{ fund.name }}
+            </p>
+
+            <p class="fund-card__desc">
+              {{ fund.shortDescription }}
+            </p>
+
+            <div class="fund-card__progress-bar">
+              <div
+                class="fund-card__progress"
+                :style="`width: ${getCapProgress(fund)}%`"
+              />
+            </div>
+
+            <p class="fund-card__funded">
+              <!-- eslint-disable max-len -->
+              {{ 'funds.fund-card-funded' | globalize({ funded: fund.currentCap / fund.hardCap }) }}
+              <!-- eslint-enable max-len -->
+            </p>
+
+            <p class="fund-card__invested">
+              <!-- eslint-disable max-len -->
+              {{ 'funds.fund-card-invested' | globalize({ invested: { value: fund.currentCap, currency: fund.defaultQuoteAsset } }) }}
+              <!-- eslint-enable max-len -->
+            </p>
+
+            <p class="fund-card__days-to-launch">
+              <!-- eslint-disable max-len -->
+              {{ 'funds.fund-card-days-to-launch' | globalize({ days: fund.daysToGo }) }}
+              <!-- eslint-enable max-len -->
+            </p>
+
+            <vue-markdown
+              class="fund-card__offer"
+              :source="'funds.fund-card-offer' | globalize({
+                baseHardCap: {
+                  value: fund.baseHardCap,
+                  currency: fund.baseAsset
+                },
+                hardCap: {
+                  value: fund.hardCap,
+                  currency: fund.defaultQuoteAsset
+                }
+              })"
+              :html="false"
+            />
+          </div>
+        </a>
+      </div>
+    </template>
+
+    <template v-else-if="isLoaded">
+      <no-data-message
+        icon-name="inbox"
+        :msg-title="'funds.no-funds-title' | globalize"
+        :msg-message="'funds.no-funds-desc' | globalize"
+      />
+    </template>
+
+    <template v-else>
+      <loader :message-id="'funds.loading-msg'" />
+    </template>
+
+    <collection-loader
+      v-show="filteredFunds.length"
+      class="funds__loader"
+      :first-page-loader="recordsLoader"
+      @first-page-load="setRecords"
+      @next-page-load="extendRecords"
+    />
   </div>
 </template>
 
 <script>
 import TopBar from '@/vue/common/TopBar'
 import Drawer from '@/vue/common/Drawer'
+import Loader from '@/vue/common/Loader'
+import CollectionLoader from '@/vue/common/CollectionLoader'
+import NoDataMessage from '@/vue/common/NoDataMessage'
+
+import SelectField from '@/vue/fields/SelectField'
 import CreateFundForm from '@/vue/forms/CreateFundForm'
+import FundShortDetails from '@/vue/pages/funds/FundShortDetails'
```

Comment:
the word details is very common and doesn't give us any information about the component content

Reply:
Renamed to `SaleOverview`